valhalla/baldr/directededge.h

```diff
@@ -1118,10 +1152,12 @@ class DirectedEdge {
   uint64_t tunnel_ : 1;         // Is this edge part of a tunnel
   uint64_t bridge_ : 1;         // Is this edge part of a bridge?
   uint64_t traffic_signal_ : 1; // Traffic signal at end of the directed edge
+  uint64_t stop_sign_ : 1;      // Stop sign at end of the directed edge
+  uint64_t yield_sign_ : 1;     // Yield/give way sign at end of the directed edge
```

Comment:
these have to go at the end where the spare was. by putting them here you break backwards compatibility. old code loading new tiles would get the value of stop sign for seasonal and the value of yield sign for deadend and bss connection would be seasonal. we cant shift the values down inside the 64bit integer

Reply:
@kevinkreiser my bad.  I will fix

Reply:
@kevinkreiser updated.